pages/journal.js

```diff
@@ -3,16 +3,320 @@ import Helmet from 'react-helmet';
+      this.setState({
+        activeTab: index,
+      });
+    }
+
+    const tabs = () => {
+      let tabs = [{title: 'Spring 2015'}, {title: 'Spring 2014'}, {title: 'Spring 2013'}, {title: 'Spring 2012'}, {title: 'Fall 2010'}];
+      const HTMLify = function (item, i) {
+        return (
+			<div
+			  key={i}
+			  onClick={() => handleClick(i)}
+			  className = "tab"
+			>
+			  {item.title}
+			</div>
+        );
+      }
+      return (
+		<div className="tab-nav">
+		  {tabs.map(HTMLify)}
+		</div>
+	  );
+    }
+
+    const tabContent = () => {
+      const activeTab = this.state.activeTab;
+      const data = tabData[activeTab];
+	  const content = function (item) {
+		return (
+		  <div>
+			<div className="author">
+			  {item.author}
+			</div>
+			
+			<div className="title">
+			  {item.title}
+			</div>
+		  </div>
+		);
+	  }
+      return (
+        <div className="tab-content">
+		  <div className="tab-title">
+		    {data.title}
+		  </div>
+		  
+		  	<img className="cover-img" src={data.cover} />
+
+		  
+		  {data.content.map(content)}
+		</div>
+      );
+    }
+	
     return (
       <div className="">
         <Helmet
           title={config.siteTitle}
         />
-        <h1>
-          Journal
-        </h1>
-      </div>
+		
+		<div className="content-header">
+          <span>Journal / </span>
+          <span className="subheader">Recent Publications</span>
+        </div>
+		
+		<div className="journals">
+		  {tabs()}
+		  {tabContent()}
+		</div>
+		
+        <div className="submit">
+		  <div className="content-header">
+            <span>Journal / </span>
+            <span className="subheader">Submit</span>
+          </div>
+		
+		  <div className="question">
+			What is the Berkeley Student Journal of Asian Studies?
+		  </div>
+		  <div className="answer">
+			The Berkeley Student Journal of Asian Studies (BSJAS) is an Institute of East Asian Studies sponsored journal that strives both to showcase the best undergraduate and graduate work at Berkeley in the field of Asian Studies and to give students aspiring to a future in academia a chance to practice the process of submission to a journal.
+		  </div>
+
+		  <div className="question">
+			Who is eligible to submit an entry?
+		  </div>
+		  <div className="answer">
+			We encourage submissions from graduate and undergraduate students currently enrolled at the University of California, Berkeley, or that have graduated within the past 12 months. Submissions can be from any discipline, as long as they are related to Asia and are original research.
+		  </div>
+
+		  <div className="question">
+			When is the submission deadline?
+		  </div>
+		  <div className="answer">
+			The current deadline is January 16, 2016.
+		  </div>
+
+		  <div className="question">
+			What is the submission process?
+		  </div>
+		  <div className="answer">
+			The applicant first submits a publication for selection under the review of the Journal Committee.<br/><br/>
```

Comment:
Either use paragraph tags or div tags with a bottom margin, try to avoid using double br tags for spacing